sources/packages/backend/workflow/src/deploy.ts

```diff
@@ -114,7 +116,7 @@ dotenv.config({ path: path.join(__dirname, "../../../../.env") });
 
     console.info(`\nDeployment successful!\n`);
     console.info(
-      "Please see below the definitions deployed (non-modified definitions will not generate a new version)."
+      "Please see below the definitions deployed (non-modified definitions will not generate a new version).",
```

Comment:
why is the "comma , " appended in this file in many lines? did our lint settings change?

Reply:
Apparently yes. This file was not receiving changes for a while. On my VS Code (using this PR branch), it was formatted also adding the comma.
Please feel free to give it a try on your local also and check 😉